aws_lambda.py

```diff
@@ -56,6 +64,8 @@ def validate_signature(event, context):
 
         # 1. Grab collection information
         dest_col = client.get_collection()
+        if 'signature' not in dest_col['data']:
+            continue
```

Comment:
Hang on, doesn't this sort of defeat the purpose of the lambda (by allowing unsigned collections to go through)?

Reply:
haha, yes indeed. Instead of this, I should probably check the signer capabilities!